projects/ngrx.io/content/guide/eslint-plugin/overview.md

```diff
@@ -0,0 +1,76 @@
+# Overview
+
+## Rules
```

Comment:
What do you think about moving "Rules" section to a new page? We could use the content from ["Recipes => ESLint NgRx Plugin"](https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/guide/eslint-plugin.md) page for "ESLint Plugin => Overview" & "ESLint Plugin => Installation" pages.

Reply:
@markostanimirovic as discussed, I added the intro text to the overview page :)